### PR TITLE
Cs 5599

### PR DESCRIPTION
**Pull request by thnkloud9**

### CODE REVIEW

File: newscoop/src/Newscoop/PluginGeneratorBundle/Resources/skeleton/bundle/composer.json.twig [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "name": "newscoop/{{ pluginName | lower }}-plugin-bundle",
```
Comment (ahilles107) [inline newscoop/src/Newscoop/PluginGeneratorBundle/Resources/skeleton/bundle/composer.json.twig]: vendor should be genrated also

**Comment by ahilles107** [comment]
:+1:

File: newscoop/src/Newscoop/PluginGeneratorBundle/Resources/skeleton/bundle/composer.json.twig (hunk 1 above)
Comment (thnkloud9) [inline newscoop/src/Newscoop/PluginGeneratorBundle/Resources/skeleton/bundle/composer.json.twig]: ah, yeah, good point.  I can add a 'vender' input option to the generator command and go through and find out how many places that will need to be updated.  In plugins dir that would mean the path would be:
  plugins/{vender}/{pluginName} 

namespace would then be:
  {vender}/{pluginName}PluginBundle

and bundle:
  {vender}{pluginName}PluginBundle

does that sound right?

Comment (ahilles107) [inline newscoop/src/Newscoop/PluginGeneratorBundle/Resources/skeleton/bundle/composer.json.twig]: yep

**Comment by ahilles107** [comment]
please rebase it to one descriptive commits, and let's merge it. We will need also instructions how to use it on dev docs.

**Comment by ahilles107** [comment]
@thnkloud9 ping!

**Comment by thnkloud9** [comment]
closing due to poorly attempted commit squashing.

**Pull request closed without merging**